src/librbd/cache/Block.h

```diff
@@ -0,0 +1,69 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#ifndef CEPH_LIBRBD_CACHE_BLOCK_MAP
+#define CEPH_LIBRBD_CACHE_BLOCK_MAP
+
+#include "librbd/cache/Types.h" 
+#include "include/lru.h"
+#include <mutex>
+#include <unordered_map>
+
+namespace librbd {
+namespace cache {
+ 
+struct Entry : public LRUObject {
+  uint64_t on_disk_id;
+  Entry() : on_disk_id(0) {}
+};
+
+struct Block {
+  std::mutex lock;
+
+  uint64_t block;
+  Entry* entry;
+  uint8_t status;
+
+  void *tail_block_io_request; ///< used to track ios on this block
```

Comment:
Is there any special use of adding 3 backslashes for comments.
Normally we use 2.